File: dart/transaction.dart
enum TransactionType { deposition, withdrawal }

class Transaction {
  late DateTime date;
  late TransactionType type;
  late double amount;

  Transaction(this.amount, this.type) {
    this.date = DateTime.now();
  }

  @override
  toString() {
    return "${date.toIso8601String()} || $amount Ar || $type";
  }
}

File: python/transaction.py
from datetime import datetime

class Transaction:
    def __init__(self, amount, type):
        self.date = datetime.now()
        self.amount = amount
        self.type = type

    def __str__(self):
        return f'{self.date.isoformat()} || {self.amount}Ar || {self.type}'